playground/mocks/config/responseConfig.js

```diff
@@ -835,6 +835,12 @@ const secondaryEmail = {
   ]
 };
 
+const selfServiceRegistration = {
+  '/api/v1/registration/form': [
+    'registration-form'
+  ]
+};
+
 module.exports = {
-  mocks: idx,
+  mocks: selfServiceRegistration,
```

Comment:
May need to revert this change. I assume you edited for local testing but I think `idx` mocks have been the default scenario configured.

Reply:
Yeah that won't get merged. I usually leave a note to self DONKTMERGE. Just forgot this time